public/library-interior-script.js

```diff
@@ -260,3 +353,45 @@ const goBack = () => {
     pageTransitionFunc("/index.html");
   }, 2000);
 };
+
+const wantToRead = (bookNum) => {
```

Comment:
One could refactor this by merging `wantToRead` and `notInterested`, with a parameter to pass `key` value